app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="login_form">
```

Comment:
- Kindly follow the design guidelines for the Login page:
    - [ ] Kindly add the `Back link` in the header which redirects to the previous page.
    - [ ] Please add the `login link` in the navbar in the header responsible for the login process instead of adding a login button.
   - [ ] Kindly remove the input's label and use placeholders instead.

| Your design | Guidelines |
| --- | --- |
| ![image](https://github.com/MNisarAli/budget_trackie/assets/24703171/efc01723-74ab-4234-963e-d2f9b6789697) | ![image](https://user-images.githubusercontent.com/24703171/219885103-63122090-4832-4d4f-b31f-0fc2e80b3777.png) |

Reply:
Hi :wave: @gemmen29,
Thank you so much for your suggestion i have updated the app styles per the design guidelines for the login page.